src/test/java/io/phdata/streamliner/schemadefiner/DiffGeneratorTest.java

```diff
@@ -0,0 +1,110 @@
+package io.phdata.streamliner.schemadefiner;
+
+import io.phdata.streamliner.schemadefiner.ConfigBuilder.DiffGenerator;
+import io.phdata.streamliner.schemadefiner.model.Configuration;
+import io.phdata.streamliner.schemadefiner.model.ConfigurationDiff;
+import io.phdata.streamliner.schemadefiner.util.StreamlinerUtil;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.Assert.*;
+
+public class DiffGeneratorTest {
```

Comment:
In this test, we are checking that we are getting diffs, which is good. 

Can we either modify these tests or add new ones which test for a single column and then verify the output is exactly what we expect.

For example:

1. Test a single column add, then verify that is the only output we get.
2. Test for a single column delete, then verify the same.
3. Test for a single column change, then verify the same.